src/lib/tenant/tenantSlugFactory.ts

```diff
@@ -13,7 +13,7 @@ export default class TenantSlugFactory {
         return "ETHERFI";
 
       case TENANT_NAMESPACES.UNISWAP:
-        return "UNISWAP";
+        return "UNI";
```

Comment:
@andreitr fyi I updated slug to UNI, please check if this breaks anything on FE. This is how it's defined on the DB